the_hack_series/resources/FACILITATION.md

```diff
@@ -0,0 +1,135 @@
+# Hack Series Facilitation
+
+Here is the schedule for Hack Series. We'd highly recommend going through
+a test run through the entire schedule.
+
+## Schedule
+
+_Italisized_ schedule items are things that only facilitators do
+
+**Bolded** schedule items are things that both hackers and facilitators do<kbd></kbd>
+
+Time  | Activity
+------|-----------------------------------------
+11:00 | _Figure out who is doing what_
+12:00 | _Setup_
+12:45 | _Play Host & Facilitate Conversation_
+1:10  | _Hack Series Introduction & Icebreakers_
+1:20  | _Introduce Part I_
+1:25  | **Part I: Follow the step-by-step workshop**
+2:30  | **Pizza Time**
+3:00  | _Introduce Part II_
+3:05  | **Part II: Customize your website**
+3:40  | _Collect Demo Links in tlk.io_
+3:45  | **Demos!**
+3:55  | _Collect Feedback_
+4:00  | **Finish**
```

Comment:
I know that our table format for Markdown files isn't very standardized, but in other places where we use tables we border them with `|`. Can you change this to the following?

```
|   Time | Activity                                     |
| ------ | -------------------------------------------- |
|  11:00 | _Figure out who is doing what_               |
|  12:00 | _Setup_                                      |
|  12:45 | _Play Host & Facilitate Conversation_        |
|   1:10 | _Hack Series Introduction & Icebreakers_     |
|   1:20 | _Introduce Part I_                           |
|   1:25 | **Part I: Follow the step-by-step workshop** |
|   2:30 | **Pizza Time**                               |
|   3:00 | _Introduce Part II_                          |
|   3:05 | **Part II: Customize your website**          |
|   3:40 | _Collect Demo Links in tlk.io_               |
|   3:45 | **Demos!**                                   |
|   3:55 | _Collect Feedback_                           |
|   4:00 | **Finish**                                   |
```